src/webpack/resolveModuleRules.ts

```diff
@@ -46,7 +46,10 @@ export default (isDevelopment: boolean, isLibrary: boolean): webpack.RuleSetRule
   {
     test: /\.ts(x?)$/,
     include: SOURCE_PATH,
-    use: [{ loader: 'ts-loader' }],
+    use: [{
+      loader: 'ts-loader',
+      options: { onlyCompileBundledFiles: true },
```

Comment:
I did the same thing in another PR which is pending right now. It is not merged yet because I don't have time to test it. Did you test if it works correctly in real project?

Reply:
Great! Yup, I tested building a library and using it in the app.